include/misc/wallinTerran.hpp

```diff
@@ -80,8 +80,8 @@ namespace ghost
   vector< Building > makeTerranBuildings()
   {
     // Academy
-    Building a1 = factoryTerranBuilding("Academy");
-    Building a2 = factoryTerranBuilding("Academy");
+//     Building a1 = factoryTerranBuilding("Academy");
```

Comment:
I really don't think we should change the buildings list: it works faster than Michal method with a larger space search. We should keep it! Plus, having more diverse buildings can help the gap objective, and with barracks and supplies only, the techtree objective is nonsense.